src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Effects/BevelBitmapEffect.cs

```diff
@@ -50,7 +50,6 @@ unsafe protected override SafeHandle CreateUnmanagedEffect()
         [Obsolete(MS.Internal.Media.VisualTreeUtils.BitmapEffectObsoleteMessage)]
         protected override void UpdateUnmanagedPropertyState(SafeHandle unmanagedEffect)
```

Comment:
we should be able to remove this method entirely now, yeah?

Reply:
this goes for all the `UpdateUnmanagedPropertyState` methods in this PR

Reply:
This is an implementation of an abstract method of the public parent class `BitmapEffect`. https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.effects.bitmapeffect?view=netframework-4.8 (which I am hesitant to remove because of the build breaks that might cause).

Reply:
gotcha!